config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1950,6 +1950,10 @@
 # DEFAULT: false
 # gradebookng.allowColumnResizing=true
 
+# S2U-26: Turn on multiple gradebook instances for groups
+# DEFAULT: false
+# gradebookng.multipleGroupInstances=true
```

Comment:
```suggestion
# gradebook.byGroup.enabled=true
```

Reply:
I understand this property is to enable this feature globally but do individual sites get to choose to use this feature?

Reply:
Yes, that is right. Sites can enable or disable it when adding/editing tools, in the same screen as other multi-instance tools like Lessons or LTI.